test/wcsmo14/demos/truss/truss_compliance_2d1.jl

```diff
@@ -1,16 +1,16 @@
 module TrussComplianceDemo2D1
 
 using TopOpt, LinearAlgebra, StatsFuns
-# using Makie, GLMakie
-# using TopOpt.TrussTopOptProblems.TrussVisualization: visualize
+using Makie, GLMakie
```

Comment:
CI doesn't have access to Makie and GLMakie so we usually comment these lines out. They work on your computer but when I tried before it wasn't easy to get GLMakie working on CI. Please check that these lines run on your computer and then comment them out.

Reply:
By "these lines" I mean the visualization lines.

Reply:
All the documentation scripts run as part of CI to ensure they are up to date so this is not just documentation, CI actually tries running all this code.

Reply:
> CI doesn't have access to Makie and GLMakie so we usually comment these lines out. They work on your computer but when I tried before it wasn't easy to get GLMakie working on CI. Please check that these lines run on your computer and then comment them out.

Ok